Tests/AztecTests/Importer/InAttributeConverterTests.swift

```diff
@@ -1,6 +1,10 @@
 import XCTest
-@testable import Aztec
 import libxml2
+#if SWIFT_PACKAGE
+@testable import HTMLParser
+#else
+@testable import Aztec
+#endif
```

Comment:
This one is even more confusing to me… why do we import one package in one context (`#if SWIFT_PACKAGE`) and a different one in another? 🤔